python/ray/dashboard/client/package.json

```diff
@@ -31,6 +31,7 @@
     "usehooks-ts": "^2.9.4"
   },
   "devDependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```

Comment:
do you add the json file accidentally?

Reply:
Hi Kevin, it looks like it was added by accident. I’ve removed it in my next commit.